wiki-service/src/main/java/org/exoplatform/wiki/service/impl/WikiServiceImpl.java

```diff
@@ -377,7 +378,21 @@ public Wiki createWiki(String wikiType, String owner) throws WikiException {
     wikiPreferences.setWikiPreferencesSyntax(wikiPreferencesSyntax);
     wiki.setPreferences(wikiPreferences);
     Wiki createdWiki = dataStorage.createWiki(wiki);
-
+    StringBuilder sb = new StringBuilder("= Welcome to ");
+    String spaceName = owner;
+    if(wikiType.equals(PortalConfig.GROUP_TYPE)) {
+      try {
+        spaceName = getSpaceNameByGroupId(owner);
+      } catch(Exception e) {
+        if (LOG.isDebugEnabled()) {
+          LOG.debug("Can't get Space name by group ID : " + owner, e);
+        }
+      }
+      
+    }
+    sb.append(spaceName).append(" =");
+    createdWiki.getWikiHome().setContent(sb.toString());
+    updatePage(createdWiki.getWikiHome(), PageUpdateType.EDIT_PAGE_CONTENT);

```

Comment:
Use updatePage(createdWiki.getWikiHome(), null) instead. No need to broadcast the event (or it will create an activity and we don't want that).